Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1445,7 +1445,7 @@
         };
     };
 
-    $.Autocompleter.Selection = function(field, start, end) {
+    $.Autocompleter.Selection = function(field, start, end, focusInputAfterSelection) {

```

Comment:
okay, that's strange. it doesn't look like this is ever called with false. is this correct?

Reply:
`function autoFill(query, sValue) {` (l. 616) calls `Selection`:
```javascript
$.Autocompleter.Selection(input, query.length, query.length + sValue.length);
```

Reply:
Should false be passed in explicitly here? I thought it makes sense to leave the value to default here.

Reply:
Oh well, need to check one more time if this works in IE anyway.

Reply:
no, please pass true/false everywhere. otherwise, it's hard to do a good review and later on read the code. Plus, undefined !== false in Javascript

Reply:
Alright, will do.